website/views.py

```diff
@@ -366,3 +366,7 @@ def redirect_about(**kwargs):
 
 def redirect_howosfworks(**kwargs):
     return redirect('/getting-started/')
+
+
+def redirect_meetings_analytics_link(**kwargs):
+    return redirect('/getting-started/?utm_source=osf4m&utm_medium=email&utm_campaign=success')
```

Comment:
@whit537, I'm double checking with @saradbowman to see if the analytics links given in the JIRA ticket are correct. I feel like `successadd#addons` at the end of the second link should just be `success#addons`, in which case a second redirect wouldn't be necessary. I'm also assuming that the first link shouldn't have `#addons` at the end.